src/errors/runtime/index.js

```diff
@@ -170,3 +170,12 @@ export class SkipJsErrorsArgumentApiError extends APIError {
         super('skipJsErrors', code, ...args);
     }
 }
+
+export class ImportESMInCommonJSError extends GeneralError {
+    constructor (originalError, targetFile) {
+        //NOTE: There is different error message for 14.* node and 16.* and higher.
+        const [, esModule] = originalError.toString().match(/ES Module:? (\S*)/);
```

Comment:
`Node.14x` will be outdated in the next year (`2023-04-30`).

I propose to add conditional logic here to simplify removing outdated code in the future.

```js
_getESModule(err) {
     const regExp = semver.gt(process.version, '14) ? // : //

    ....
}

 const [, esModule] = this._getESModule(originalError);
```